src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj

```diff
@@ -106,6 +106,7 @@
           DependsOnTargets="DownloadDnnModelFiles">
     <Message Importance="High" Text="Copying external model files... @(ModelFileCopy->'%(TargetPath)')" />
     <Copy SourceFiles="@(ModelFileCopy)"
+          Condition="'$(SkipRIDAgnosticAssets)' == ''"
```

Comment:
Condition checking in MSBuild is a little different. Typically "boolean" properties will have 3 values: empty, true, and false. So generally, you pick empty (the default) to be considered the same as either 'true' or 'false'. In our case here, the default for `SkipRIDAgnosticAssets` should be `false`. So instead it is best to write this condition as:

`Condition="'$(SkipRIDAgnosticAssets)' != 'true'"`

That way someone can explicitly set `/p:SkipRIDAgnosticAssets=false`, and it still works correctly.